cli/src/lib.rs

```diff
@@ -93,6 +94,9 @@ pub struct Iroha {
     thread_handlers: Vec<ThreadHandler>,
     /// Relay that redirects messages from the network subsystem to core subsystems.
     _sumeragi_relay: AlwaysAddr<FromNetworkBaseRelay>, // TODO: figure out if truly unused.
+    /// A boolean value used to stop the peers ability to recieve data from the network. Used in
+    /// sumeragi testing.
+    pub freeze_status: Arc<AtomicU32>,
```

Comment:
I'm not fan of tendency to include testing specific variables into the primary modules. 
Maybe we can add some kind of wrapper that will drop messages?

Reply:
@Erigara I agree. It's a sign of a bad design. Sometimes it can't be helped

Reply:
not if it has to be cross-crate. Feature flag of compile flag would help

Reply:
I will put hide it under debug-assertions such that it does not get compiled in when we build for release. I do think however that we should clearly divide what is "public" and what is not. It is not appropriate to apply the same standards of public interfaces that are a part of our API that goes to customers to application internal interfaces. The internal interfaces should only be governed by principles that aid speed of development and producing the best product.

Reply:
Actually I doubt we can do better than `debug_assertions` recall that we will need to test the debug build using @astrokov7 's tests in Python.